python/common/org/python/ImportLib.java

```diff
@@ -79,7 +79,7 @@
                 if (!name.equals("*")) {
                     try {
                         if (native_import) {
-                            java.lang.Class java_class = java.lang.Class.forName(java_name.toString().replace("/", ".") + name);
+                            java.lang.Class java_class = java.lang.Thread.currentThread().getContextClassLoader().loadClass(java_name.toString().replace("/", ".") + name);
```

Comment:
I take it that this is required to make sure that, during testing, the non-caching class loader is in use? Does this effectively mean that any use of `Class.forName()` will fail under test?